src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/adaptive_pooling/adaptive_pooling_kernel_ref.cpp

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "adaptive_pooling_kernel_ref.h"
+#include "kernel_selector_utils.h"
+#include <algorithm>
+#include <string>
+
+namespace kernel_selector {
+
+ParamsKey AdaptivePoolingRef::GetSupportedKey() const {
+    ParamsKey k;
+    k.EnableInputDataType(Datatype::INT32);
+    k.EnableInputDataType(Datatype::INT64);
```

Comment:
Int data type is actually supported in kernel? From the spec of the op, only floating point is supported. It the data type is supported, it should be tested in unit test/single layter test.

Reply:
you are right the first input is float always, but we also have second indices input whis is Int. That's why we need to support it as input type also, right?

Reply:
Got it! Thanks!